### DISCUSSION TITLE
How to read a file

### DISCUSSION THREAD

**Discussion by lanttu1243** [vlang/v#12089] [General]
Hello,
I would like to know how can I read a file in V.

**Comment by JalonSolov** [comment]
The online module docs, https://modules.vlang.io/, has a search box.

There are a LOT of different read routines - you'll need to pick the one that makes the most sense for your task.

**Comment by larpon** [comment]
For small to medium text files this is a nice oneliner:
`contents := os.readfile(filename) or { panic(err) }`
